xtend-contrib/src/main/java/de/oehme/xtend/contrib/Builder.xtend

```diff
@@ -54,6 +67,15 @@ class BuilderProcessor extends AbstractClassProcessor {
 					'''
 					primarySourceElement = cls
 				]
+				addMethod('''set«builtField.simpleName.toFirstUpper»''') [
```

Comment:
I just tried and it works for me. This method
```(xtend)
def Builder name(String name) {
  this.name = name
  this
}
```

can be accessed from the outside like this

```
new Builder => [
  name = "Foo"
]
```